frontends/web/src/api/exchanges.ts

```diff
@@ -63,18 +63,31 @@ export const getMoonpayBuyInfo = (code: string) => {
   };
 };
 
+export type ExchangeErrorCode = 'userAbort' | 'addressNotFound';
+
 export type AddressSignResponse = {
-  status?: 'success' | 'abort';
+  success: boolean;
   signature: string;
   address: string;
-  error: string;
+  errorMessage?: string;
+  errorCode?: ExchangeErrorCode;
 }
```

Comment:
I think that `maybeBB02Err` is only used to map call results that go through bb02 API. It would be nice to refactor the generic endpoint communication using a standard Response struct with those three fields to be used (and extended when needed). But I guess this is a pretty broad change, that requires one or more dedicated PRs